wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -161,39 +164,68 @@ class HomeActivity : ComponentActivity(), HomeView {
                         state = scalingLazyListState
                     ) {
                         if (inputBooleans.isNotEmpty()) {
-                            items(inputBooleans.size) { index ->
-                                if (index == 0)
-                                    SetTitle(R.string.input_booleans)
-                                SetEntityUI(inputBooleans[index], index)
+                            item {
+                                SetListHeader(
+                                    id = R.string.input_booleans,
+                                    expanded = entityViewModel.expandedInputBooleans
+                                )
+                            }
+                            if (entityViewModel.expandedInputBooleans) {
+                                items(inputBooleans.size) { index ->
+                                    SetEntityUI(inputBooleans[index], index)
+                                }
                             }
                         }
```

Comment:
These should use state hoisting: https://developer.android.com/jetpack/compose/state#state-hoisting

Reply:
ok i think its already doing that now with the proper values? it does work :)

Reply:
one other thing to consider here is that `item {}` needs to be used within the lazy list itself. I can't move it out to another composable to handle it.